src/main/java/Duke.java

```diff
@@ -1,10 +1,54 @@
+import java.util.Scanner;
+
 public class Duke {
+
```

Comment:
It would be better to write some comments to illustrate what are the functions of the methods